web/core/models.py

```diff
@@ -83,3 +90,29 @@ class Meta:
 
     def __str__(self):
         return self.name
+
+
+def get_file_name_storage(instance, filename):
+    return Path(instance.kid.id) / filename
+
+
+class KidImage(Model):
+    kid = ForeignKey(Kid, on_delete=CASCADE, verbose_name="criança")
+    image = FileField(
+        verbose_name="Foto",
+        upload_to=get_file_name_storage,
+        validators=[
+            FileExtensionValidator(allowed_extensions=["jpg", "jpeg", "png", "gif"])
+        ],
+    )
+
+    @property
+    def image_render(self):
+        return mark_safe(f'<img src="{self.image.url}" width="150" height="150" />')
```

Comment:
Esse código está quebrando **sem** o S3. Seria legal se a gente pudesse desenvolver (localmente) sem o S3. Será que tem como?

Reply:
Eita. Esquece! Era barbeiragem minha!

Reply:
Sim. Podemos usar `DEBUG` quando for `False` ou criar algo como `AWS_ENABLED=False` O que acha melhor ?